internal/osbuild2/sshd_config_stage.go

```diff
@@ -0,0 +1,20 @@
+package osbuild2
+
+type SshdConfigConfig struct {
+	PasswordAuthentication          bool `json:"PasswordAuthentication"`
+	ChallengeResponseAuthentication bool `json:"ChallengeResponseAuthentication"`
+	ClientAliveInterval             int  `json:"ClientAliveInterval"`
```

Comment:
All of these values should be marked as 'omitempty` and most probably also defined as pointer values, because the "zero" value for the type would otherwise end up in the stage options and would be defined in the SSHD configuration.

Reply:
No problem, what could go wrong if all the variables are pointers :-)